src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -20,6 +23,45 @@ var AppComponent = React.createClass({
       "text-warning": isDeploying
     });
 
+    var healthData = [
```

Comment:
It's not model logic. But you are right that it does not belong to AppComponent but into the AppHealthComponent.